File: CS/CustomDialogs/MySearchForm.cs
#region #CustomSearchTextForm
using System.Drawing;
using DevExpress.XtraRichEdit.Forms;

namespace CustomDialogs
{
    public partial class MySearchTextForm : SearchTextForm
    {
        public MySearchTextForm(SearchFormControllerParameters controllerParameters, string searchWord)
            : base(controllerParameters)
        {
            lblFndDirection.Location = new Point (lblFndDirection.Location.X - 10, lblFndDirection.Location.Y);
            lblFndDirection.Text = "Direction:";
            cbFndSearchString.Text = searchWord;
            chbFndRegex.Enabled = false;
        }

    }
}
#endregion #CustomSearchTextForm

File: VB/CustomDialogs/MySearchForm.vb
#Region "#CustomSearchTextForm"
Imports System.Drawing
Imports DevExpress.XtraRichEdit.Forms

Namespace CustomDialogs
	Partial Public Class MySearchTextForm
		Inherits SearchTextForm

		Public Sub New(ByVal controllerParameters As SearchFormControllerParameters, ByVal searchWord As String)
			MyBase.New(controllerParameters)
			lblFndDirection.Location = New Point(lblFndDirection.Location.X - 10, lblFndDirection.Location.Y)
			lblFndDirection.Text = "Direction:"
			cbFndSearchString.Text = searchWord
			chbFndRegex.Enabled = False
		End Sub

	End Class
End Namespace
#End Region ' #CustomSearchTextForm